GetF!t/ContentView.swift

```diff
@@ -7,10 +7,11 @@ struct ContentView: View {
         ZStack {
             GradientBackground()
             TabView(selection: $selectedTabManager.selectedTab) {
-                WelcomeView()
+                WelcomeView(welcomeViewModel: WelcomeViewModel.init())
                     .tag(9)
                 ForEach(0 ..< Exercise.exercises.count) { index in
-                    ExerciseView(index: index)
+                    // TODO: predelat ViewModel
+                    ExerciseView(exerciseViewModel: ExercisesViewModel.init(index: 0))
```

Comment:
celé toto content view by som trochu prerobila. Všimla som si že aj v exercise view aj vo welcome view dávaš navrch HeaderView. Prerobíme to tak, že ho dáš do ContentView:
```swift
ZStack {
            GradientBackground()
            VStack {
                // tu si všimni že header view už nemá v inite view model (to som ti písala v inom komentári)
                // v inite nepredávam ani title, ten bude v SelectedTabManager
                HeaderView()
                TabView { ... }
            } 
```

Dosiahneme tým lepší UX, bude to vyzerať krajšie a zároveň nám to zjednoduší kód 🙂 

Aby to fungovalo musíš prerobiť `SelectedTabManager` aby mal published property title. Niečo na tento štýl:
```swift
class SelectedTabManager: ObservableObject {
    @Published var selectedTab = 9
    @Published var title: LocalizedStringKey = LocalizedStringProvider.WelcomePage.welcome
    
    func goToWelcomeView() {
        selectedTab = 9
        title = LocalizedStringProvider.WelcomePage.welcome
    }

   ...
```

Reply:
done